todo_app/data/trello_items.py

```diff
@@ -94,4 +94,33 @@ def complete_item(card_id):
 
     response = requests.request("PUT", reqUrl, params=query_params)
     
+    print(response.text)
+def doing_item(card_id):
+
+
+    reqUrl = f"https://api.trello.com/1/cards/{card_id}/"
+
+    query_params = {
+        "key": trello_key,
+        "token": trello_token,
+        "idList": "626653880ecf1c46be8073f9"
```

Comment:
I think these might be best set as environment variables :) These are unique for every board, so we'll want to be able to change these from person to person